draft-deshpande-rats-multi-verifier.md

```diff
@@ -95,6 +96,15 @@ It may not be economically viable to build and maintain such a degree of complex
 5. The mix today is a combination of Verifier services provided by component manufacturers, Verifiers provided by integrators, and Verifiers under local authority (i.e., close to the attester). Rarely is it just one of these.
 
 
+# Scope of the Document
+When Multiple Verifiers are required for Appraisal, then it is important to standardize the topology of the Verifiers and also
+information or messages exchanged between each of the Verifier in the eco-system.
+
+This enables inter-operatibility among different Verifiers owned by different Verifier Owners.
```

Comment:
```suggestion
This enables inter-operatibility among different Verifiers owned by different Verifier Owners.
Otherwise absence of standardisation can lead to fragmentation of the eco-system.
```